dbt_platform_helper/templates/env/overrides/stack.ts

```diff
@@ -23,18 +21,9 @@ export class TransformedStack extends cdk.Stack {
         this.appName = props.appName;
         this.envName = props.envName;
 
-        this.addLogResourcePolicy();
```

Comment:
I think to support existing 100% AWS Copilot applications,  we need to keep this, but make it conditional on whether there is a `terraform` directory?